### ISSUE TITLE
AccessibilityView should be Identifiable

### ISSUE THREAD

**Issue by AgapovOne** [VODGroup/VoiceOverDesigner#272]
Identifiable support would allow use of ForEach in SwiftUI.

Also it's just good to be able to identify element or container somehow.

Label is not an id, frame too.

**Comment by akaDuality** [comment]
Also will be useful for artboard to make relation between button and presented screen

**Comment by AgapovOne** [comment]
Tried a concept, but `[any AccessibilityView]` array can't be Identifiable because of Swift limitations.

Possible workaround is [AnyAccessibilityView], but then I should wrap everything in it and then unwrap to somehow compare like if `A11yDescription() == AnyAccessibilityView`.

Tried to use `AccessibilityViewCast` instead of `[any AccessibilityView]`, but that's not convenient too.

Have to choose something, will investigate more
